hydra/test_utils/test_utils.py

```diff
@@ -309,12 +317,16 @@ def experiment(cfg):
 
     config_name = ""
     if task_config is not None:
-        cfg_file = tmpdir / "config.yaml"
+        cfg_file = conf_dir / "config.yaml"
+        with open(str(cfg_file), "w") as f:
+            f.write("# @package _global_\n")
+            OmegaConf.save(task_config, f)
+        cfg_file = modified_tmpdir / "config.yaml"
```

Comment:
1. To simplify reviewing, please reorganize your diff to do save the config under the conf directory in a standalone commit.

2. Primary config files (like this one) does not require a package header.